.gitignore

```diff
@@ -10,3 +10,19 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# jetbrains ide stuff
+.idea*
+
+.swp*
+.swo*
+
+
+bin/
+vendor/github.com/
```

Comment:
Don't you want this to be part of the commit? Or does the vendor.json ensure everything gets pulled down?

Reply:
vendor.json will pull everything with a "govendor sync" command, so we dont need to store any 3rd party source in our repo